check-requirements.in

```diff
@@ -1,7 +1,16 @@
 bandit
+darglint
 flake8
+flake8-annotations-complexity
+flake8-bugbear
+flake8-broken-line
+flake8-commas
 flake8-comprehensions
+flake8-expression-complexity
 flake8-import-order
+flake8-quotes
+flake8-rst-docstrings
+flake8-simplify
```

Comment:
General code style hinting at simplifications (like checking error conditions before and above a function's happy flow).